weblate/accounts/views.py

```diff
@@ -1339,10 +1339,13 @@ def setup(self, request, *args, **kwargs):
             self.sort_query = "-date_joined"
 
     def get_context_data(self, **kwargs):
+        search = self.form.cleaned_data.get("q", "").strip()
+
```

Comment:
Please move this below the method comment. Or avoid the intermediate variable completely as it is used in single place only.